src/screens/tezos-token-screen/tezos-token-screen.tsx

```diff
@@ -6,9 +6,8 @@ import { HeaderCard } from '../../components/header-card/header-card';
 import { PublicKeyHashText } from '../../components/public-key-hash-text/public-key-hash-text';
 import { TokenEquityValue } from '../../components/token-equity-value/token-equity-value';
 import { TokenScreenContentContainer } from '../../components/token-screen-content-container/token-screen-content-container';
-import { loadActivityGroupsActions } from '../../store/activity/activity-actions';
 import { loadSelectedBakerActions } from '../../store/baking/baking-actions';
-import { loadTezosBalanceActions } from '../../store/wallet/wallet-actions';
+import { loadActivityGroupsActions, loadTezosBalanceActions } from '../../store/wallet/wallet-actions';
```

Comment:
unused import

Reply:
You may be wondered but it is used: https://github.com/madfish-solutions/templewallet-mobile/blob/b600dbb1e0b2ce33402d4c8b4dfc4f10f55a91b0/src/screens/tezos-token-screen/tezos-token-screen.tsx#L23